### ISSUE TITLE
NVDA does not announce the number of sheets when opening Google Sheets or Excel files

### ISSUE THREAD

**Issue by Anand1990-afk** [nvaccess/nvda#18558]

**Comment by seanbudd** [comment]
As requested in https://github.com/nvaccess/nvda/issues/18485#issuecomment-3086593475 please fill out the issue template in full.
